etna/lib/commands.rb

```diff
@@ -3,88 +3,125 @@
 require 'rollbar'
 require 'tempfile'
 require_relative 'helpers'
+require 'yaml'
 
-class EtnaCommands
-  class Help < Etna::Command
-    usage 'List this help'
+# /bin/etna will confirm execution before running any command that includes this module.
+module RequireConfirmation
+end
 
-    def execute
-      puts 'Commands:'
-      EtnaApp.instance.commands.each do |name, cmd|
-        puts cmd.usage
-      end
-    end
+class EtnaApp
+  def self.config_file_path
+    File.join(Dir.home, 'config.yml')
```

Comment:
Should we use `etna.yml` to try and avoid conflicts with other software? `config.yml` seems generic enough that something else might use that filename, too.

Reply:
Oh right.  Yeah I don't know how I accidentally did that, this should be etna.yml